src/OpenTelemetry.SemanticConventions/SemanticConventionsAttributes.cs

```diff
@@ -0,0 +1,5448 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+// <auto-generated>This file has been auto generated from scripts/templates/SemanticConventionsAttributes.cs.j2</auto-generated>
+
+#pragma warning disable CS1570 // XML comment has badly formed XML
+
+namespace OpenTelemetry.SemanticConventions
+{
+    /// <summary>
+    /// Constants for semantic attribute names outlined by the OpenTelemetry specifications.
+    /// </summary>
+    /// <remarks>
+    /// Schema and specification version: https://opentelemetry.io/schemas/1.25.0.
+    /// </remarks>
+    public static class SemanticConventionsAttributes
```

Comment:
`OpenTelemetry.SemanticConvetions.SemanticConventionsAttributes` seems excessive. If we decide to keep all attributes in one place, I'd suggest `OpenTelemetry.SemanticConvetions.Attributes` (`OpenTelemetry.SemanticConvetions.Metrics`, `OpenTelemetry.SemanticConvetions.Events`, etc...)